src/main/java/ch/njol/skript/effects/EffContinue.java

```diff
@@ -52,26 +55,41 @@
 			"\t\tcontinue # only print when counter is 1, 2, 3, 5 or 10",
 		"\tbroadcast \"Game starting in %{_counter}% second(s)\"",
 })
-@Since("2.2-dev37, 2.7 (while loops)")
+@Since("2.2-dev37, 2.7 (while loops), INSERT VERSION (outer loops)")
 public class EffContinue extends Effect {
 
 	static {
-		Skript.registerEffect(EffContinue.class, "continue [loop]");
+		Skript.registerEffect(EffContinue.class, "continue [[this|[the] (%*integer%(st|nd|rd|th)|current)] loop]");
 	}
```

Comment:
I think something like this might be a slightly better design, not only fixes the issue that was brought up about `continue loop` being a breaking change currently but seems like a little bit easier design to read this way anything using the number can be treated as it's respectful format
```suggestion
		Skript.registerEffect(EffContinue.class,
			"continue [[this|the [current]] loop]",
			"continue [the] %*integer%(st|nd|rd|th) loop"
		);
	}
```